styles/style.css

```diff
@@ -151,3 +151,38 @@ section {
   width: 87%;
   margin: auto;
 }
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 40%;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.527);
+  backdrop-filter: blur(8px);
+}
+
+.close {
```

Comment:
perhaps we want to change the cursor to a pointer to indicate the x is clickable?